runtime/vm/resolvesupport.cpp

```diff
@@ -360,7 +380,10 @@ resolveClassRef(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA cpIndex, UDAT
 	 * 		- the current class and resolved class are identical
 	 * 		- the current class was loaded by the bootstrap class loader
```

Comment:
The comment needs an update for the new condition

Reply:
The comment updated.